lib/core/src/lib/form/components/widgets/date/date.widget.spec.ts

```diff
@@ -197,13 +199,13 @@ describe('DateWidgetComponent', () => {
             fixture.detectChanges();
 
             let dateButton = element.querySelector<HTMLButtonElement>('button');
-            expect(dateButton.disabled).toBeFalsy();
+            expect(dateButton?.disabled).toBeFalsy();
```

Comment:
Maybe instead of 
```
expect(dateButton?.disabled).toBeFalsy();
```
we should have 
```
expect(dateButton).toBeDefined();
expect(dateButton.disabled).toBeFalsy();
```
It should cover the case of when dateButton is undefined/null. I tried running the following test case, and it passed
```
    fit('temp unit test', () => {
       let obj = null;
       expect(obj?.disabled).toBeFalsy();
       obj = undefined;
       expect(obj?.disabled).toBeFalsy();
    });
```

Reply:
thanks, will try... it's not about test actually, but the most recent typescript flagging it was "obj is potentially null"

Reply:
the ESLint converts "expect(dateButton && dateButton.disabled)" to the same short form I used